modules/nw-metallb-configure-l2-advertisement-interface.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-interface_{context}"]
+= Configuring MetalLB with a L2 advertisement by using interface
```

Comment:
I'm not exactly sure what the right answer is, but "by using interface" sounds like it's missing something.

Should it be "by using interfaces" or "by setting interfaces"?

Reply:
After a hard time thinking of an alternative, I see  'Configuring MetalLB with a L2 advertisement for selected interfaces' to be a better version.